main/main.c

```diff
@@ -659,7 +659,40 @@ PHPAPI ZEND_COLD void php_log_err_with_severity(char *log_message, int syslog_ty
 	if (PG(error_log) != NULL) {
 #ifdef HAVE_SYSLOG_H
 		if (!strcmp(PG(error_log), "syslog")) {
-			php_syslog(syslog_type_int, "%s", log_message);
+			static char nvt[] = " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~";
```

Comment:
@pprindeville this seems just fine. Debian Jessie with locale all `en_US.UTF-8`.

```
$ cat syslog.test
#!/bin/bash

logger "füße абракадабра 芬蘭"
```
```
$ file syslog.test
syslog.test: Bourne-Again shell script, UTF-8 Unicode text executable
```

```
$ tail /var/log/messages
......
Aug  7 21:26:57 PHP-Anatol-1 anatol: füße абракадабра 芬蘭
```

So far it seems UTF-8 has to be handled. The basic algo seems to be splitting a message by LF/CRLF.

Regarding the security concerns - could you give an example please? In general, it would mean the syslog API itself were insecure. Not sure to follow this part.

Thanks.